src/main/java/com/unfamilia/eggbot/domain/raidpackage/Item.java

```diff
@@ -5,13 +5,25 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 
+import javax.persistence.ManyToOne;
+
 @Data
 @EqualsAndHashCode(callSuper = false)
 public class Item extends PanacheEntity {
     private String name;
-    private ItemCategory itemCategory;
+    @ManyToOne(optional = false)
+    private ItemCategoryOption itemCategory;
```

Comment:
So a Category can have multiple items but then in line 20 it has another item category?

Reply:
Whoopsie